tests/importing/test_load_from_mgf.py

```diff
@@ -11,14 +11,3 @@ def test_load_from_mgf_using_filepath():
 
     assert len(spectra) > 0
     assert isinstance(spectra[0], Spectrum)
-
-
-def test_load_from_mgf_using_file():
-    module_root = os.path.join(os.path.dirname(__file__), "..")
-    spectra_filepath = os.path.join(module_root, "testdata", "pesticides.mgf")
-
-    with open(spectra_filepath, "r", encoding="utf-8") as spectra_file:
```

Comment:
This is now the standard way of loading, so this test does not work anymore. See #555 Do you agree that this is expected behavior? @florian-huber @hechth

Reply:
Yes, this should indeed be closed as you pointed out in #555.